bip-musig2.mediawiki

```diff
@@ -549,6 +549,15 @@ There are, however, insecure toy schemes with AGM security proofs, but those exp
 In contrast, in MuSig2, all group elements that arise in the protocol are known to the adversary and declared as group element inputs.
 A scheme very similar to MuSig2 and with two-point nonces was independently proven secure in the ROM and AGM by [https://eprint.iacr.org/2020/1245 Alper and Burdges].
 
+== Change Log ==
+
+To help implementors understand updates to this BIP, we attach a version number that resembles ''semantic versioning'' (<code>MAJOR.MINOR.PATCH</code>).
+A change in the <code>MAJOR</code> version indicates that the specification is incompatible with prior versions.
+The <code>MINOR</code> version is increased whenever the inputs or the output of an algorithm changes in a backward compatible way.
```

Comment:
When do we increase `PATCH`? For *every* other change, or are we allowed to rephrase things without a patch? 

For example: Do we increase `PATCH` if
 * we fix a typo?
 * improve an explanation?
 * change an algorithm without changing its I/O behavior?

What do we change if we rename an algorithm?

We don't perfect answers here. This is not actual semantic versioning but we should have some guidelines in mind.

I found this by the way: https://datatracker.ietf.org/doc/html/draft-claise-semver-02

Reply:
Good find! I had thought we increment it whenever we do something that's should get the attention of implementors. So to answer your questions: no, depends, depends. Added a sentence for that and harmonized with the ietf spec. Or should we just refer to it?

Reply:
sounds good and pragmatic